test_e2e.sh

```diff
@@ -162,6 +162,7 @@ function finish() {
 trap finish INT EXIT
 
 SLOW_GIT=/slow_git.sh
```

Comment:
Can you also rename slow_git.sh to slow_git_clone.sh ?

Reply:
done.